be/src/vec/exprs/table_function/vexplode.cpp

```diff
@@ -44,10 +44,9 @@ VExplodeTableFunction::VExplodeTableFunction() {
 
 Status VExplodeTableFunction::_process_init_variant(Block* block, int value_column_idx) {
     // explode variant array
-    const auto& variant_column = check_and_get_column<ColumnObject>(
-            remove_nullable(block->get_by_position(value_column_idx)
-                                    .column->convert_to_full_column_if_const())
-                    .get());
+    auto column_without_nullable = remove_nullable(block->get_by_position(value_column_idx).column);
+    auto column = column_without_nullable->convert_to_full_column_if_const();
+    const auto& variant_column = check_and_get_column<ColumnObject>(column.get());
```

Comment:
better use use assert_cast to instead check_and_get_column

Reply:
done